commands/group_test.go

```diff
@@ -0,0 +1,114 @@
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestTeamGroupListCmd() {
+
+	s.Run("Team group list with no existed user returns error", func() {
```

Comment:
It's the team that is missing. I would rephrase into `Team group list returns error when passing a nonexistent team`